job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext.groovy

```diff
@@ -297,4 +297,31 @@ class TriggerContext extends AbstractExtensibleContext {
             }
         }
     }
+
+    /**
+     * Configures the stash-pullrequest-builer plugin.
+     */
+    @RequiresPlugin(id = 'stash-pullrequest-builder')
+    void stashPullRequestTrigger(@DslContext(StashPullRequestContext) Closure closure = null) {
```

Comment:
Add a `@since 1.38` JavaDoc tag and add a minimum version to the `@RequiresPlugin` annotation.